package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "calculator",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "big.js": "^5.2.2",
```

Comment:
This is not the correct dependency, the dependency listed in the Pathwright is `big-js`; you might probably have stickler error using `big.js`. You can import it as `npm i big-js`.

![image](https://user-images.githubusercontent.com/31897434/77975322-b0f8c680-72f1-11ea-97df-20e14f60578b.png)


![image](https://user-images.githubusercontent.com/31897434/77975270-827aeb80-72f1-11ea-8d9c-f41f7d5cebf7.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/